providers/standard/src/airflow/providers/standard/operators/latest_only.py

```diff
@@ -74,6 +80,15 @@ def choose_branch(self, context: Context) -> str | Iterable[str]:
             now,
         )
 
+        if left_window == right_window:
+            self.log.info(
+                "Zero-length interval [%s, %s) from timetable (%s); treating current run as latest.",
+                left_window,
+                right_window,
+                self.dag.timetable.__class__,
+            )
+            return list(context["task"].get_direct_relative_ids(upstream=False))
```

Comment:
This is because we changed the default to `DeltaTriggerTimetable` from `DeltaDataIntervalTimetable` for  3.0 in https://github.com/apache/airflow/pull/46846